ui-library-quickstart-composites/src/App.tsx

```diff
@@ -1,30 +1,46 @@
-import { AzureCommunicationTokenCredential } from '@azure/communication-common';
+import { AzureCommunicationTokenCredential, CommunicationUserIdentifier } from '@azure/communication-common';
 import {
   CallComposite,
-  CallAdapter,
-  createAzureCommunicationCallAdapter,
   ChatComposite,
-  ChatAdapter,
-  createAzureCommunicationChatAdapter
+  fromFlatCommunicationIdentifier,
+  useAzureCommunicationCallAdapter,
+  useAzureCommunicationChatAdapter
 } from '@azure/communication-react';
-import React, { useEffect, useMemo, useState } from 'react';
+import React, { CSSProperties, useEffect, useMemo, useRef, useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { ChatClient } from '@azure/communication-chat';
 
-function App(): JSX.Element {
-  // Common variables
-  const endpointUrl = '<Azure Communication Services Resource Endpoint>';
-  const userId = '<Azure Communication Services Identifier>';
-  const displayName = '<Display Name>';
-  const token = '<Azure Communication Services Access Token>';
-  // Calling Variables
-  // Provide any valid UUID for `groupId`.
-  const groupId = '<Developer generated GUID>';
-  // Chat Variables
-  const threadId = '<Get thread id from chat service>';
+/**
+ * Authentication information needed for your client application to use
+ * Azure Communication Services.
+ *
+ * For this quickstart, you can obtain these from the Azure portal as described here:
+ * https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/identity/quick-create-identity
+ *
+ * In a real application, your backend service would provide these to the client
+ * application after the user goes through your authentication flow.
+ */
+const ENDPOINT_URL = '<Azure Communication Services Resource Endpoint>';
+const USER_ID = '<Azure Communication Services Identifier>';
+const TOKEN = '<Azure Communication Services Access Token>';
+
+/**
+ * Display name for the local participant.
+ * In a real application, this would be part of the user data that your
+ * backend services provides to the client application after the user
+ * goes through your authentication flow.
+ */
+const DISPLAY_NAME = '<Display Name>';
 
-  const [callAdapter, setCallAdapter] = useState<CallAdapter>();
-  const [chatAdapter, setChatAdapter] = useState<ChatAdapter>();
+/**
+ * Entry point of your application.
+ */
+function App(): JSX.Element {
+  // Arguments that would usually be provided by your backend service or
+  // (indirectly) by the user.
+  const { endpointUrl, userId, token, displayName, groupId, threadId } = useAzureCommunicationServiceArgs();
 
-  // We can't even initialize the Chat and Call adapters without a well-formed token.
+  // A well-formed token is required to initialize the chat and calling adapters.
   const credential = useMemo(() => {
```

Comment:
Note for other pr reviewers:
credential is created is here. (was collapsed for me).